docs/src/getting_started/using_cmaize_as_build_system/obtaining_cmaize.rst

```diff
@@ -0,0 +1,68 @@
+****************
+Obtaining CMaize
+****************
+
+The easiest and most common way to use CMaize in your project is by
+:ref:`Automatically Downloading and Including CMaize`. Alternatively users can
+:ref:`Download CMaize Manually`.
+
+Automatically Downloading and Including CMaize
+==============================================
+
+As a convenience to your users you can make it so that your build system
+automatically downloads CMaize and includes it. The easiest way to do this is
+to put the following CMake script in a file ``cmake/get_cpp.cmake``:
```

Comment:
makes sense to change the name to `get_cmaize.cmake`

Reply:
I updated this in the example projects and docs.